Automattic/wp-calypso
# Replace /block-editor with /themes?status=active endpoint

#### Proposed Changes

* The `/sites/${ siteSlug }/block-editor` endpoint was replaced with the `/sites/${ siteSlug }/themes?status=active` endpoint in order to retrieve the value representing if a theme is FSE enabled. This new endpoint will return the data that will help us determine if a site's theme is FSE enabled. These changes affect the new/old site setup flows

#### Testing Instructions



* Navigate to http://calypso.localhost:3000/start/setup-site/design-setup-site?siteSlug={siteSlugHere}, click 'Start building', select a theme and ensure that you are navigated to the `/home` page. Check the `/themes` page to ensure the correct theme has been selected.
* Navigate to http://calypso.localhost:3000/setup/designSetup?siteSlug={siteSlugHere}, select any free theme, ensure that you are navigated to the `/home` page. Check the `/themes` page to ensure the correct theme has been selected.

#### Pre-merge Checklist






Related to #65697
